docs/Developer-Guide.md

```diff
@@ -292,6 +292,7 @@ $ popd
 >   variable in the previous command and ensure the `qemu-img` command is
 >   available on your system.
 >   - If `qemu-img` is not installed, you will likely see errors such as `ERROR: File /dev/loop19p1 is not a block device` and `losetup: /tmp/tmp.bHz11oY851: Warning: file is smaller than 512 bytes; the loop device may be useless or invisible for system tools`. These can be mitigated by installing the `qemu-img` command (available in the `qemu-img` package on Fedora or the `qemu-utils` package on Debian).
+> - If `loop` module is not probed, you will likely see errors such as `losetup: cannot find an unused loop device`. Execute `modprobe loop` could resolve it.
```

Comment:
Please fix the indentation.

Reply:
Usually the loop module is provided as built-in in most distros. So this should really not be required. You can mention is as "If the loop module support (CONFIG_BLK_DEV_LOOP) is not built into the kernel, the loop module will need to be loaded...."